app/main.py

```diff
@@ -1,54 +1,34 @@
-import json
-import xml.etree.ElementTree as ET
+from __future__ import annotations
 
+from app.book import Book
+from app.print import ConsolePrint, ReversePrint
+from app.display import ConsoleDisplay, ReverseDisplay
+from app.serializers import JsonSerialize, XmlSerialize
 
-class Book:
-    def __init__(self, title: str, content: str):
-        self.title = title
-        self.content = content
 
-    def display(self, display_type: str) -> None:
-        if display_type == "console":
-            print(self.content)
-        elif display_type == "reverse":
-            print(self.content[::-1])
-        else:
-            raise ValueError(f"Unknown display type: {display_type}")
-
-    def print_book(self, print_type: str) -> None:
-        if print_type == "console":
-            print(f"Printing the book: {self.title}...")
-            print(self.content)
-        elif print_type == "reverse":
-            print(f"Printing the book in reverse: {self.title}...")
-            print(self.content[::-1])
-        else:
-            raise ValueError(f"Unknown print type: {print_type}")
+def main(book: Book, commands: list[tuple[str, str]]) -> None | str:
+    displays = {
+        "console": ConsoleDisplay(),
```

Comment:
No need to create a class just use it as type
Fix in all places

Reply:
Don't understand what do you mean